### PR TITLE
Update to kubernetes-1.15.4

### PR DESCRIPTION
**Pull request by sslavic**
Towards giantswarm/giantswarm#6302

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -42,14 +42,14 @@
...
+  version = "kubernetes-1.15.4"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```
Comment (sslavic) [inline Gopkg.toml]: This is to be reverted once https://github.com/giantswarm/helmclient/pull/119 gets merged.

File: Gopkg.toml [hunk 2]
```diff
@@ -65,5 +65,33 @@
   name = "github.com/giantswarm/microkit"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```
Comment (sslavic) [inline Gopkg.toml]: This is to be reverted once https://github.com/giantswarm/operatorkit/pull/319 gets merged.

File: main.go [hunk 3]
```diff
@@ -27,11 +28,7 @@ import (
 )
 
 var (
-	description string     = "draughtsman is an in-cluster agent that handles Helm based deployments."
-	f           *flag.Flag = flag.New()
-	gitCommit   string     = "n/a"
-	name        string     = "draughtsman"
-	source      string     = "https://github.com/giantswarm/draughtsman"
+	f *flag.Flag = flag.New()
```
Comment (sslavic) [inline main.go]: These and related changes are due to breaking changes in `microkit` and `microendpoint` made back in July (see Slack [thread](https://gigantic.slack.com/archives/C04TGHDEF/p1563961136007200)).

File: Gopkg.toml [hunk 4]
```diff
@@ -65,5 +65,33 @@
...
   name = "github.com/giantswarm/operatorkit"
+
+[[override]]
```
Comment (sslavic) [inline Gopkg.toml]: These overrides enforce version of transitive dependencies to be consistent with the direct ones; without them `dep ensure -update` would either not complete or would pick wrong (typically master) version/branch.

File: main.go (hunk 3 above)
Comment (rossf7) [reply main.go]: Ack. Good catch and draughtsman doesn't get updated very often.

**Review by rossf7 — APPROVED** [review]
LGTM when tested

**Pull request merged**